.travis.yml

```diff
@@ -84,6 +84,12 @@ jobs:
       name: 'End to End Tests'
       script:
         - unbuffer node build-system/pr-check/e2e-tests.js
+      env:
+        - CACHE_NAME=E2EJOB
```

Comment:
Pretty sure these two lines should be removed. (Were they from a bad merge?)

Reply:
i think it was from a bad rebase on my end

Reply:
removed.